src/Provider/GeocodeEarth/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
+
+## 4.0.0
```

Comment:
Not sure what this should be, let me know if it should be different.

Reply:
Yes, I know it's a bit confusing 🤣 

It's a new provider, you can start at `1.0.0`.

Reply:
Ok, will do tomorrow :)